src/app/App.tsx

```diff
@@ -1,46 +1,30 @@
 import React from 'react';
+import { Route, BrowserRouter as Router } from 'react-router-dom';
 import styled from 'styled-components';
+import firebase from '../firebase/firebase';
 import { CreateContainer } from '../create/CreateContainer';
-import { Error } from '../Error';
-import { Loading } from '../Loading';
-import { GridContainer, GridContainerProps } from '../grid/GridContainer';
+import { GridLoaderContainer } from '../grid/GridLoaderContainer';
 import { Footer } from '../shared/Footer';
 
+firebase.init();
+
 const AppLayout = styled.div`
   flex-direction: column;
   display: flex;
   width: 100%;
   height: 100%;
 `;
 
-export interface AppProps {
-  loading: boolean;
-  gridId: string;
-  grid?: GridContainerProps;
-}
-
-function withFooter(component: React.ReactNode) {
+const App: React.FunctionComponent = () => {
   return (
-    <AppLayout>
-      {component}
-      <Footer />
-    </AppLayout>
+    <Router>
+      <AppLayout>
+        <Route exact path="/" component={CreateContainer} />
```

Comment:
Consider using react-router 5's hooks interface for routing related things now. No behaviour changes, but I imagine they will prefer supporting this API in future